[Challenges] Updated browser support (#12143)
cloudflare/cloudflare-docs: content/waf/reference/cloudflare-challenges.md

If your visitors encounter issues using a major browser besides Internet Explorer, they should upgrade their browser.

### Browser extensions

If you have browser extensions installed, they might lead to unpassable challenge loops. To fix, disable your extensions and reload the page.

{{<Aside type="note">}}

> This behavior occurs because some extensions modify your browser's default `User-Agent` value.

Author: ```suggestion
This behavior commonly occurs because some extensions modify your browser's default `User-Agent` value.
```

> This behavior occurs because some extensions modify your browser's default `User-Agent` value.

Author: Slightly tweaked the wording to try and help clarify that, `commonly occurs because an extensions modifies`

> This behavior commonly occurs because some extensions modify your browser's default `User-Agent` value.

Author: ```suggestion
This behavior commonly occurs because an extensions modifies your browser's default `User-Agent` value.
```

> This behavior commonly occurs because an extensions modifies your browser's default `User-Agent` value.

Reviewer: ```suggestion
This behavior commonly occurs because an extension modifies your browser's default `User-Agent` value.
```
